userland/tools/uncrustify/uncrustify.cfg

```diff
@@ -1,5 +1,111 @@
 # Uncrustify-0.65_f
 
+# ====================================================================
```

Comment:
yes, I'm generally too sloppy and perform rebasing as the last step but I guess I should be rebasing all the time i.e., I have to make it a habit!!!